ceph.spec.in

```diff
@@ -791,6 +791,16 @@ Requires:	librbd1 = %{_epoch_prefix}%{version}-%{release}
 Daemon for mirroring RBD images between Ceph clusters, streaming
 changes asynchronously.
 
+%package -n ceph-dedup
+Summary:	Ceph daemon and tool for deduplication
+%if 0%{?suse_version}
+Group:		System/Filesystems
+%endif
+Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
+Requires:	librados2 = %{_epoch_prefix}%{version}-%{release}
+%description -n ceph-dedup
+Daemon for selective deduplication, targeting a Ceph pool.
```

Comment:
I'm not sure whether we actually want a package for this.  It's certainly not mature enough for usage in a real cluster, and we do not want to mislead users.